src/common/options/mds.yaml.in

```diff
@@ -516,6 +516,17 @@ options:
   services:
   - mds
   with_legacy: true
+- name: mds_log_segment_size_factor
+  type: float
+  level: advanced
+  desc: ratio of max(MDLog's segment size, maximum size of every 10000 events) at which
+    the MDLog will start a new LogSegment immediately if the maximum size of the 10000
+    events is larger than half of the sigment size. Set a value less than or equal to 1
+    to disable it.
+  default: 2
+  services:
+  - mds
+  with_legacy: true
```

Comment:
nit: typo
there are quite a few places where **segment** has been written as **sigment**
please check the commit message as well

Reply:
Sure, will fix it. Thanks @mchangir